stake-pool/program/src/processor.rs

```diff
@@ -493,6 +567,11 @@ impl Processor {
             &vote_account_address,
         )?;
 
+        if meta.lockup != stake_program::Lockup::default() {
+            msg!("Invalid lockup exists on validator stake account");
+            return Err(StakePoolError::WrongStakeState.into());
```

Comment:
We could have a more descriptive error here than "Wrong" I think

Reply:
Yeah, there are still a few renames to undertake...